### ISSUE TITLE
Mean and std values for S2 images used for pretraining

### ISSUE THREAD

**Issue by faurefrancois** [zhu-xlab/Copernicus-FM#4]
Hi,

Thank you for this interesting model!
I didn't find it in the paper or in the code, is it possible to have the mean and std values of Sentinel-2 images used for pretraining the model?
(I am using R, G, B and NIR for finetuning the model, that's why I ask).

Thank you!

**Comment by faurefrancois** [comment]
Ok I think I have the answer, maybe I didn't look well at first sight ^. When going to finetuning S2 dataset configuration files, we can see those values:

band_stats:
  mean: [1353.7,1117.2,1041.8,946.5,1199.1,2003.0,2374.0,2301.2,2599.7,732.1,1820.6,1118.2]
  std: [897.3,736.0,684.8,620.0,791.9,1341.3,1595.4,1545.5,1750.1,475.1,1216.5,736.7]

As it is said in the paper that the normalization values are recommended to be the same as the pretraining dataset, I guess that those are the normalization values of the training dataset.
